CarpetLib/src/copy_4d.cc

```diff
@@ -101,13 +101,12 @@ void copy_4d(T const *restrict const src, ivect4 const &restrict srcpadext,
   ptrdiff_t const dstloff = dstoff[3];
 
   // Loop over region
-  if (use_openmp) {
+  if (false and use_openmp) {
```

Comment:
this should not be committed in this form. Rather the whole "if" branch should be removed.